jwt/jwt_test.go

```diff
@@ -1 +1,673 @@
 package jwt
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/cap/oidc"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+// TestValidator_Validate_Valid_JWT tests cases where a JWT is expected to be valid.
+func TestValidator_Validate_Valid_JWT(t *testing.T) {
+	tp := oidc.StartTestProvider(t)
```

Comment:
the reuse of the oidc.TestProvider makes me giggle.

Reply:
It was very helpful 👍